litmus/PTX/Proxy-SingleThread-rf-surW-surF-conF-conR.litmus

```diff
@@ -0,0 +1,15 @@
+PTX Proxy-SingleThread-rf-surW-surF-conF-conR
+"rf over different non-generic proxies with both proxy fences in correct order are enough, the result will be guaranteed"
+"the generic memory is synchronize with surface cache by surface fence, then constant fence synchronize the data from generic memory to constant cache. Hence the cold is guaranteed to rf sust"
+{
+rd1 = 0;
+rd2 @ generic aliases rd1;
+P0:r0=0;
+}
+ P0@cta 0,gpu 0                   ;
+ sust.weak rd1, 42                ;
+ fence.proxy.surface              ;
```

Comment:
why is this fence needed?

Reply:
I think it was because the first store is sust. This is the same scenario as one comments before I think the sust should't go with the generic location rd1. The locations of this tests are updated, another location `rd3 @ surface aliases rd1` is added. The surface-fence is going to synchronize the write from rd3 to rd1, then the constant fence synchronize from rd1 to rd2.